src/TestSuite/TestCase.php

```diff
@@ -197,21 +197,22 @@ public function loadFixtures()
      * Useful to test how plugins being loaded/not loaded interact with other
      * elements in CakePHP or applications.
      *
-     * @param array $plugins list of Plugins to load
+     * @param array $plugins List of Plugins to load.
      * @return \Cake\Http\BaseApplication
      */
     public function loadPlugins(array $plugins = [])
     {
+        /** @var BaseApplication $app */
```

Comment:
Technically this is `|MockObject`
But as long as the mock is not used inside of the method, this is fine.

Reply:
Should use FQCN.

Reply:
@dereuromark agree, but for this I focused on readability and on the actual relevant use.
@ADmad done.